### ISSUE TITLE
collection-list builder provides confusing selection feature

### ISSUE THREAD

**Issue by martenson** [galaxyproject/galaxy#3827]
You can 'select' rows by clicking on them turning them dark. But it has no effect on what datasets will be in the list and is confusing in this case. Possibly a remnant of list-pairs builder where this has a use since you are 'pairing' datasets.

![screenshot 2017-03-27 17 09 38](https://cloud.githubusercontent.com/assets/1814954/24378364/abece200-1310-11e7-9edf-57ee8a695c3c.png)

**Comment by scrathat** [comment]
@martenson can this and #4970 be closed?

**Comment by martenson** [comment]
@scrathat I don't think so, both issues are still present and as a bonus I found that the black selection now hides the discard button.

![Galaxy___Data_Libraries](https://user-images.githubusercontent.com/1814954/67724259-dab16880-f9b4-11e9-8e1d-bad105e6a56e.png)

**Comment by bernt-matthias** [comment]
Still the same on 23.0
